tests/install_upgrade_operators/product_upgrade/test_eus_upgrade.py

```
@@ -29,14 +33,14 @@ def test_eus_upgrade_process(
         upgraded_odf,
         non_eus_to_target_eus_ocp_upgraded,
         non_eus_to_target_eus_cnv_upgraded,
-        eus_created_target_hco_csv,
+        created_target_hco_csv,
         eus_unpaused_workload_update,
```

Ruff (0.15.15)

[warning] Unused method argument: `eus_unpaused_workload_update`

(ARG002)